inngest/src/main/kotlin/com/inngest/Introspection.kt

```diff
@@ -0,0 +1,44 @@
+package com.inngest
+
+import com.beust.klaxon.Json
+
+abstract class Introspection(
+    open val functionCount: Int,
+    open val hasEventKey: Boolean,
+    open val hasSigningKey: Boolean,
+    open val mode: String,
+    @Json("authentication_succeeded") open val authenticationSucceeded: Boolean?,
+    @Json("schema_version") val schemaVersion: String = "2024-05-24",
```

Comment:
is it possible to use `@JsonNaming(PropertyNamingStrategy.SnakeCaseStrategy.class)` so you don't have to manually snake case every property? I found this on https://stackoverflow.com/questions/10519265/jackson-overcoming-underscores-in-favor-of-camel-case

Reply:
great suggestion i was really hoping for it to work but it doesn't seem that it works with Klaxon and i don't think it has an equivalent for it either. I wonder if a custom converter would work though.

Reply:
oh right I missed the SO thread was about Jackson specifically. This could be nice as a follow up but non blocking for this PR